tests/core/gui/test_revoked_user.py

```diff
@@ -13,14 +15,25 @@ async def test_revoked_notification(
     central_widget = logged_gui.test_get_central_widget()
     assert central_widget is not None
 
+    # Make sure we're connected
+    def _connected():
+        assert central_widget.connection_status == BackendConnStatus.READY
+
+    assert central_widget.connection_status == BackendConnStatus.READY
+    await aqtbot.wait_until(_connected)
+
+    # Revoke the user
     await backend.user.revoke_user(
         organization_id=alice.organization_id,
         user_id=bob.user_id,
         revoked_user_certificate=b"dummy",
         revoked_user_certifier=alice.device_id,
     )
 
-    # Assert dialog
+    # Make sure we try to reach the backend
+    await aqtbot.run(central_widget.show_users_widget)
```

Comment:
Why is this necessary ? The event listener connection should close as soon as the revoked event is processed in the backend